src/app/items/item-by-id.component.html

```diff
@@ -1,76 +1,92 @@
 <ng-container *ngIf="state$ | async as state">
   <ng-container *ngIf="state.isReady && state.data as itemData">
-
-    <ng-container>
-      <alg-item-header [itemData]="itemData" *ngIf="!(fullFrameContent$ | async)"></alg-item-header>
-
-      <alg-access-code-view
-        *ngIf="showAccessCodeField$ | async"
-        i18n-sectionLabel sectionLabel="Access to activity"
-        i18n-buttonLabel="Button label for joining an item by (group) code" buttonLabel="Access"
-        [itemData]="itemData"
-        (groupJoined)="reloadItem()"
-      ></alg-access-code-view>
-
-      <alg-item-permissions
-        *ngIf="watchedGroup$ | async as watchedGroup"
-        [itemData]="itemData"
-        [watchedGroup]="watchedGroup"
-        (changed)="reloadItem()"
-      ></alg-item-permissions>
-
-      <alg-tab-bar *ngIf="!!(shouldDisplayTabBar$ | async) && !(fullFrameContentDisplayed$ | async)" />
-      <div
-        class="content-container"
-        #contentContainer
-        *ngrxLet="currentTab$ as currentTab"
-      >
-        <ng-container *ngIf="currentTab?.isTaskTab || (itemContentComponent?.isTaskLoaded$ | async) || currentTab?.tag === 'alg-content' || currentTab?.tag === 'alg-children-edit'">
-          <alg-error *ngIf="answerLoadingError$ | async as error; else noAnswerLoadingError">
-            <ng-container *ngIf="!error.fallbackLink; else fallback" i18n>Unable to load the answer</ng-container>
-            <ng-template #fallback>
-              <ng-container i18n>
-                Unable to load the answer, <a [routerLink]="error.fallbackLink" class="alg-link">back to the regular task page</a>.
+    <ng-container *ngrxLet="isWatching$ as isWatching">
+      <ng-container *ngrxLet="fullFrameContentDisplayed$; let fullFrameContentDisplayed">
```

Comment:
Be consistent, either `x$ as x` or `x$; let x`.

To match their doc, it seems better to use `x$ as x`

Reply:
Fixed